plugin/trino-redshift/src/main/java/io/trino/plugin/redshift/RedshiftPlugin.java

```diff
@@ -13,13 +13,18 @@
  */
 package io.trino.plugin.redshift;
 
-import io.trino.plugin.jdbc.JdbcPlugin;
+import com.google.common.collect.ImmutableList;
+import io.trino.spi.Plugin;
+import io.trino.spi.connector.ConnectorFactory;
 
 public class RedshiftPlugin
-        extends JdbcPlugin
+        implements Plugin
 {
-    public RedshiftPlugin()
+    public static final String REDSHIFT = "redshift";
```

Comment:
This constant is used only from RedshiftConnectorFactory. Inline.

Reply:
It was suggested to make move the constant here. I'll inline https://github.com/trinodb/trino/pull/24117#discussion_r1855939274